Program.cs

```diff
@@ -1,10 +1,181 @@
-﻿namespace Task1
+﻿using System;
+
+namespace Task1
 {
     internal class Program
     {
+        static void DisplayArray(int size, int[] array)
+        {
+            for (int i = 0; i < size; i++)
+            {
+                if (i < (size - 1))
+                {
+                    Console.Write($"{array[i]},");
+                }
+                else
+                {
+                    Console.Write($"{array[i]}]\n");
+                }
+            }
+        }
+
+        static bool ConvertToDuodecimal(int decimalNumber)
+        {
+            string duodecimal = "";
+            char duodecimalDigit;
+
+            while (decimalNumber > 0)
+            {
+                int remainder = decimalNumber % 12;
+
+                if(remainder < 10)
+                {
+                    duodecimalDigit = (char)(remainder + '0');
+                }
+                else
+                {
+                    duodecimalDigit = (char)(remainder - 10 + 'A');
+                }
+
+                duodecimal = duodecimalDigit + duodecimal;
+
+                if(duodecimal.Contains("AA"))
+                {
+                    return true;
+                }
+
+                decimalNumber /= 12;
+            }
+
+            return false;
+        }
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello, World!");
+            //Task 1.1
+            Console.WriteLine("Please enter number a:");
+            string inputA = Console.ReadLine();
+            Console.WriteLine("Please enter number b:");
+            string inputB = Console.ReadLine();
+
+            int numberA = int.Parse(inputA);
+            int numberB = int.Parse(inputB);
+
```

Comment:
Unclear, you have to add comment why 130

Reply:
Modified